lock/src/main/java/org/infinispan/lock/api/impl/manager/CacheHolder.java

```diff
@@ -0,0 +1,19 @@
+package org.infinispan.lock.api.impl.manager;
+
+import org.infinispan.AdvancedCache;
+import org.infinispan.lock.api.impl.entries.ClusteredLockKey;
+import org.infinispan.lock.api.impl.entries.ClusteredLockValue;
+import org.infinispan.context.Flag;
+
+public class CacheHolder {
```

Comment:
We are going to mutualise this code with counters after the merge. I inspire myself from the code having that in ming, and this CacheHolder was used in that way there,  even if it might have changed a bit since I did it. For this part, I will relook at it when we will have a 'commons' module for 'clustered data structures' : https://issues.jboss.org/browse/ISPN-8393

Reply:
I meant that you can already inline all uses of that wrapper.